epub33/rs/index.html

```diff
@@ -1984,49 +2053,83 @@ <h4>Scripting Considerations</h4>
 					publications from each other, thereby limiting access to cookies, DOM storage, etc. Examples of Web
 					APIs that are tied to the concept of "origin" include Web Storage [[WEBSTORAGE]] and
 					IndexedDB [[INDEXEDDB]], which EPUB Content Documents can interact with via scripting. Reading
-					Systems that allow users to add/remove publications from a managed library (a.k.a "bookshelf") may
+					Systems that allow users to add/remove publications from a managed library (their "bookshelf") may
 					maintain the publication’s unique origin when the publication is removed and subsequently
 					re-imported into the content library. Conversely, Reading Systems may create a new unique origin for
-					every newly-added publication.</p>
+					every newly added publication.</p>
```

Comment:
Looks like I missed this comment earlier...

What I've done is split the proposed paragraph, and the original paragraph, otherwise the part about container-constrained scripts interrupts the discussion about origin. So the original paragraph now ends with:

"Assigning a unique origin ensures that [spine-level scripts](https://www.w3.org/TR/epub-33/#sec-scripted-spine) [[EPUB-33](http://localhost:120/rs/#bib-epub-33)] are isolated from other EPUB Publications, and limits access to cookies, DOM storage, etc."

The text beginning "Examples of Web APIs that are tied to ..." now starts a new paragraph, and then after that I added the rest of your proposal:

"This specification also recommends that [container-constrained scripts](http://localhost:120/rs/#sec-scripted-container-constrained) [[EPUB-33](http://localhost:120/rs/#bib-epub-33)] not be allowed to modify the DOM of the parent EPUB Content Document and/or manipulate the containing rectangle (see [4.4 Scripting](http://localhost:120/rs/#sec-scripted-content))."

Reply:
I do not know what is happening, but I do not find these texts in the previews... :-(

Reply:
Ya, we've had this before. It's seems statically.io caches an earlier version and continues to serve it.

Here's are alternative links using githack:

- Reading Systems [preview](https://raw.githack.com/w3c/epub-specs/update/privacy-security/epub33/rs/index.html)
- Reading Systems [diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/epub-specs/epub33/rs/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://raw.githack.com/w3c/epub-specs/update/privacy-security/epub33/rs/index.html)

Reply:
Ah... Thanks, found it and it is fine indeed. Thx.

I but you may want to look at the new paragraph "This specification also...": respec does not find the reference for the container-constrainted script and reports a broken reference... (you can consider this issue as resolved once that is handles).